dataflow/src/main/java/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -164,20 +214,70 @@ public Analysis(@Nullable T transfer, int maxCountBeforeWidening, ProcessingEnvi
         this.finalLocalValues = new HashMap<>();
     }
 
-    /** The current transfer function. */
+    /**
+     * Construct an object that can perform a org.checkerframework.dataflow analysis over a control
+     * flow graph, given a transfer function.
+     *
+     * @param transfer transfer function
+     * @param maxCountBeforeWidening number of times a block can be analyzed before widening
+     */
+    @SuppressWarnings("nullness")
```

Comment:
Instead of suppressing the nullness warnings, is it enough to make the deprecated fields `env` and `types` `@Nullable`?